Formula/grass.rb

```diff
@@ -117,6 +109,12 @@ def install
       args << "--with-mysql"
     end
 
+    if MacOS.version >= :el_capitan
+      # handle stripping of DYLD_* env vars by SIP when passed to utilities;
+      # HOME env var is .brew_home during build, so it is still checked for lib
+      ln_sf "#{buildpath}/dist.x86_64-apple-darwin#{`uname -r`.strip}/lib", ".brew_home/lib"
```

Comment:
Should probably also be using `"#{ENV["HOME"]}/lib"` rather than `.brew_home` directly. IMO we make promises on `ENV["HOME"]` being stable but no promises on not changing that directory name at some future point if necessary.